src/include/daos_pool.h

```diff
@@ -75,8 +62,8 @@ struct daos_space {
 
 /** Target information */
 typedef struct {
-	/** Target type */
-	daos_target_type_t	ta_type;
+	/** padding - not used */
+	uint32_t                ta_padding;
```

Comment:
It's good that we take care of avoid breaking user binaries compiled against older headers. If user code that uses `ta_type` is compiled against the new headers, the compilation will fail. We might want to confirm if this is okay. (I think it is likely okay.)